libopenage/gui/guisys/private/gui_subtree_impl.cpp

```diff
@@ -1,8 +1,10 @@
 // Copyright 2015-2017 the openage authors. See copying.md for legal info.
 
 #include "gui_subtree_impl.h"
+#include "gui_renderer_impl.h"
 
 #include <cassert>
+#include <ciso646>
```

Comment:
Never seen this header in the wild.

If it's about the `and` and `or` operators then it's better to replace them with the C++ operators.